ui/src/app/pages/analysis/interactive-notebook.tsx

```diff
@@ -286,7 +286,10 @@ export const InteractiveNotebook = fp.flow(withUrlParams(), withCurrentWorkspace
             onCancel={() => {
               this.setState({showInUseModal: false}); }}
             onCopy={() => {this.cloneNotebook(); }}
-            onPlaygroundMode={() => {this.startPlaygroundMode(); }}>
+            onPlaygroundMode={() => {
+              this.setState({showInUseModal: false});
```

Comment:
Did the order matter here? I might have naturally written this in the reverse order (startPlayground, then hide), but I don't feel strongly.

Reply:
This ordering mirrors a similar set of commands a few lines further up in this module. The order shouldn't matter.

Reply:
Ah, I didn't see that. +1 for consistency